extlinks/organisations/models.py

```diff
@@ -43,6 +43,13 @@ def limit_by_user(self):
         else:
             return False
 
+    @property
```

Comment:
This isn't necessary for this filter. We do this for `limit_by_user()` because if we're not tracking any users for this organisation there's no use in showing the filter. We could change this logic to check if any edits have been made by bots, but that feels overcomplicated. We should just always show the checkbox.